test/functional/helpers/createFixture/clientScripts.js

```diff
@@ -16,9 +16,14 @@ import readCache from "read-cache";
 import { ClientFunction } from "testcafe";
 
 const alloyEnv = process.env.ALLOY_ENV || "int";
-
+const alloyProdVersion = process.env.ALLOY_PROD_VERSION;
```

Comment:
While technically I could have retrieved the prod version from code somewhere inside here rather retrieving it in a separate script and passing it through as an environment variable, it would change `getFixtureClientScripts` from being synchronous to asynchronous, which means we would have to `await` calls to `getFixureClientScripts`. `createFixture` is the module that calls `getFixtureClientScripts`, which means it would also have to change to be asynchronous. Each spec file calls `createFixture`, so each spec file would have to `await` `createFixture`. And since each spec file is a top-level module, we'd then have to deal with this error: `Cannot use keyword 'await' outside an async function`, which means, I believe, that we would have to wrap the code in each spec file with an IIFE. 😢  So I decided to do it this way instead.